pyproject.toml

```diff
@@ -6,11 +6,20 @@ authors = ["MujtabaNasir <mujtaba.nasir1312@gmail.com>"]
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.13"
+python = ">=3.9,<3.13"
```

Comment:
So that sphinx can be supported.
Was getting this error:
The current project's supported Python range (>=3.8,<3.13) is not compatible with some of the required packages Python requirement:
  - sphinx-autodoc-typehints requires Python >=3.9, so it will not be satisfied for Python >=3.8,<3.9

Because no versions of sphinx-autodoc-typehints match >2.2.3,<3.0.0
 and sphinx-autodoc-typehints (2.2.3) requires Python >=3.9, sphinx-autodoc-typehints is forbidden.
So, because mujtaba-charm depends on sphinx-autodoc-typehints (^2.2.3), version solving failed.